optimum/exporters/openvino/convert.py

```diff
@@ -1346,3 +1351,55 @@ def _get_encoder_decoder_stateful_models_for_export(
         decoder_export_config_with_past,
     )
     return None, models_for_export
+
+
+def _get_speecht5_tss_model_for_export(
+    model: Union["PreTrainedModel", "TFPreTrainedModel"],
+    task: str,
+    library_name: str,
+    int_dtype: str,
+    float_dtype: str,
+    preprocessors: Optional[List[Any]] = None,
+    model_kwargs: Optional[Dict] = None,
+    stateful: bool = True,
+):
+    if model_kwargs is None or "vocoder" not in model_kwargs:
+        # use the default vocoder if it is not specified
+        vocoder_id = "microsoft/speecht5_hifigan"
+    else:
+        vocoder_id = model_kwargs["vocoder"]
+
+    # prepare export config
+    export_config_constructor = TasksManager.get_exporter_config_constructor(
+        model=model, exporter="openvino", task=task, library_name=library_name
+    )
+    export_config = export_config_constructor(
+        model.config,
+        int_dtype=int_dtype,
+        float_dtype=float_dtype,
+        preprocessors=preprocessors,
+        legacy=False,
+    )
+
+    export_config.variant = "default"
+    all_variants = "\n".join([f"    - {name}: {description}" for name, description in export_config.VARIANTS.items()])
+    logger.info(f"Using the export variant {export_config.variant}. Available variants are:\n{all_variants}")
```

Comment:
by setting the variant to default we will only have the case where it's set to "with-past" so no other potential variants no ?  
```suggestion
```